index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+		href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&family=Roboto:ital,wght@0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+		rel="stylesheet">
+</head>
+<body>
+    <div class="page">
+        <div class="container">
+            <!-- HEADER SECTION -->
+            <header>
+                <ul>
+                    <li><a class="toggler-icon" href="#"> My logo</a> </li>
+                    <nav class="font-style">
+                        <li><a class="toggler-icon" href="#"><img src="images/humburger.png" alt="humburger-icon"></a>
+                        </li>
+                        <!-- <li><a class="Portfolio" href="#"> Portfolio</a></li>
+                        <li><a class="contact" href="#"> contact </a></li>
+                        <li><a class="about" href="#"> about </a></li> -->

```

Comment:
- [ ] Kindly remove all commented old pieces of code in your documents to keep your documents clean.